src/sandwich-menu/components/sandwich-menu.jsx

```diff
@@ -0,0 +1,70 @@
+import MUIDataTable from 'mui-datatables'
+import React, {useEffect, useState} from "react";
+import {Box, Grid, Toolbar, Badge, IconButton} from "@material-ui/core";
+import AddShoppingCartIcon from '@material-ui/icons/AddShoppingCart';
+import 'bootstrap/dist/css/bootstrap-grid.css.map'
+import {useOnAddCart} from "../containers/hooks/use-on-add-cart";
+import {Title} from "../containers/details/title"
+import {ShowDetails} from "../containers/details/body";
+import {columns} from "../containers/table-columns";
+import {useDispatch, useSelector} from "react-redux";
+import {getAllSandwiches} from "../actions";
+
+export const Menu = () => {
+
+    const [selected, setSelected] = useState([]);
+    const dispatch = useDispatch()
+    useEffect(() => dispatch(getAllSandwiches()),[])
+    const data = useSelector(selector)
+    const onClick = useOnAddCart(selected,setSelected, selector);
+    
+    const options = {
+        filterType: 'dropdown',
+        viewColumns:false,
+        filter: false,
+        print:false,
+        rowsSelected: selected,
+        download:false,
+        textLabels:{
+            selectedRows : {
+                text: "produs(e) selectat(e)"
+            }
+        },
+        selectableRowsOnClick: true,
+        selectableRowsHeader: false,
+        onRowSelectionChange: (currentRowsSelected, allRowsSelected, rowsSelected) => setSelected(rowsSelected),
+        expandableRows: true,
+        customToolbarSelect: () => 
+            <Toolbar>
+                <div>Adauga in cos</div>
+                <Box mr={5}>
+                    <IconButton onClick={onClick}>
+                        <Badge badgeContent={selected.length} color='secondary'>
+                            <AddShoppingCartIcon />
```

Comment:
![image](https://user-images.githubusercontent.com/23569938/109488542-60a06080-7a8e-11eb-9235-7de5f530ea6c.png)

Shouldn't the text be clickable too?

Reply:
![image](https://user-images.githubusercontent.com/23569938/109488635-7f065c00-7a8e-11eb-94a8-ce9f7abdf166.png)
Also clicking the icon while I have selected sandwiches does nothing :))))

Reply:
It saves the items in the shopping cart. If you go to the shopping-cart, in the navbar, those items will be shown

Reply:
The click on the icon adds them to the shopping-cart. To view the shopping-cart, you have to go to that page from the navbar , by clicking on the shopping-cart icon

Reply:
Also, the text is now clickable too.